app/src/main/res/values-hr-rHR/strings.xml

```diff
@@ -1,114 +1,114 @@
-    <string name="success">Success!</string>
-    <string name="app_installation_preferences">%1$s Installation Preferences</string>
-    <string name="vanced_installed">Vanced has successfully been installed! Open now?</string>
-    <string name="version">Version</string>
-    <string name="music_installed">Vanced Music has successfully been installed! Open now?</string>
-    <string name="please_be_patient">Please be patient…</string>
-    <string name="launch">Open</string>
-    <string name="welcome">Welcome</string>
+    <string name="advanced">Napredno</string>
+    <string name="app_install_files_detected">%1$s instalacijska datoteka je otkrivena!</string>
+    <string name="app_install_files_detected_summary">Upravitelj je otkrio da su sve potrebne datoteke za %1$s instalaciju pronađene. Želite li nastaviti instalaciju?</string>
+    <string name="checking_updates">Provjera ažuriranja…</string>
+    <string name="chosen_lang">Jezik: %1$s</string>
+    <string name="chosen_theme">Tema: %1$s</string>
+    <string name="chosen_version">Inačica: %1$s</string>
+    <string name="guide">Vodič</string>
+    <string name="hold_on">Zaustavi!</string>
+    <string name="installing_app">Instaliram %1$s</string>
+    <string name="magisk_vanced">Koristite Magisk/TWRP inačicu Vanceda, koja više nije podržana i ne može se ažurirati ovom aplikacijom. Uklonite ju uklanjanjem Magisk modula/koristeći TWRP Vanced deinstalator.</string>
+    <string name="miui_one_title">MIUI je otkriven!</string>
+    <string name="miui_one">Kako bi instalirali Vanced, MORATE isključiti MIUI optimizaciju u razvijateljskim postavkama. (Možete zanemariti ovo upozorenje ako koristite 20.2.20 ili noviji xiaomi.eu temeljeni ROM)</string>
+    <string name="error">Greška</string>
+    <string name="redownload">Ponovno preuzmi</string>
+    <string name="security_context">Aplikaciju obavezno preuzmite s vancedapp.com, Vanced Discord poslužitelja ili Vanced GitHuba</string>
+    <string name="success">Uspješno!</string>
+    <string name="app_installation_preferences">%1$s postavke instalacije</string>
+    <string name="vanced_installed">Vanced je uspješno instaliran! Otvorite ga odmah?</string>
+    <string name="version">Inačica</string>
+    <string name="music_installed">Vanced Glazba je uspješno instalirana! Otvorite ju odmah?</string>
+    <string name="please_be_patient">Malo pričekajte…</string>
+    <string name="launch">Otvori</string>
+    <string name="welcome">Dobrodošli</string>
     <!-- Install Page -->
-    <string name="choose_preferred_language">Choose your preferred language(s) for Vanced</string>
-    <string name="light_plus_other">Light + %1$s</string>
-    <string name="select_at_least_one_lang">Select at least one language!</string>
+    <string name="choose_preferred_language">Odaberite željeni jezika za Vanced</string>
```

Comment:
It should be:
Odaberite željeni jezik za Vanced.

"jezika" is wrong case for this sentence. It should be in nominative, but instead it is in genitive.